ci_scripts/install.sh

```diff
@@ -64,6 +65,15 @@ if [[ "$RUN_FLAKE8" == "true" ]]; then
     pre-commit install
 fi
 
+# PEP 561 compliance check
+# Assumes mypy relies solely on the PEP 561 standard
+# Also assumes mypy to be available
```

Comment:
Unfortunately, we don't install mypy and the test therefore fails. An easy fix for now would be to install mypy here as well.

Reply:
You can see this in the travis-ci log for the different builds if you expand the entry for install.sh:
```
/home/travis/miniconda/envs/testenv/bin/python: No module named mypy

Failed: PEP 561 compliance
```